home/views.py

```
@@ -0,0 +1,19 @@
+from django.shortcuts import render
+from classrooms.models import Classroom, Enrollment, ClassroomTeacher
+
+
+def home(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `home`

(ANN201)